src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,36 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class StringCalculator {
+    public int add(String input) {
+        try {
+            String[] tokens;
+            if (input.startsWith("//")) {
+                int delimiterIndex = input.indexOf("\\n");
+                String customDelimiter = input.substring(2, delimiterIndex);
```

Comment:
```suggestion
                private static final int CUSTOM_DELIMITER_START_INDEX = 2;
                
                String customDelimiter = input.substring(CUSTOM_DELIMITER_START_INDEX, delimiterIndex);
```

2나 위의 `//`, '\\n' 같은 문자들을 캐싱해서 사용한다면 역시 가독성을 더 높일 수 있을 것 같아

Reply:
오케 가독성 신경써야겠다